components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/externalopensearchchecktrigger/externalopensearchcheck.go

```diff
@@ -0,0 +1,69 @@
+package externalopensearchchecktrigger
+
+import (
+	"net/http"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/checkutils"
+	"github.com/chef/automate/lib/logger"
+)
+
+type ExternalOpensearchCheck struct {
+	log  logger.Logger
+	port string
+}
+
+func NewExternalOpensearchCheck(log logger.Logger, port string) *ExternalOpensearchCheck {
+	return &ExternalOpensearchCheck{
+		log:  log,
+		port: port,
+	}
+}
+
+func (eoc *ExternalOpensearchCheck) Run(config models.Config) []models.CheckTriggerResponse {
+	return runCheckForOpensearch(config, constants.EXTERNAL_OPENSEARCH_API_PATH, eoc.port, eoc.log)
```

Comment:
if we passing the constant from here, we can directly use this in internal method